### PR TITLE
Task/numbers - apidocs - numbersAssignment

### PR DESCRIPTION
**Pull request by xxlaykxx**
POST/GET /accounts/{accountId}/numbersAssignment
GET /accounts/{accountId}/numbersAssignment/{orderId}
Снимок экрана 2022-04-19 в 14 45 51

### CODE REVIEW

File: site/specs/numbers.json [hunk 1]
```diff
@@ -7661,8 +7661,9 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieve a list of the TelephoneNumbersAssignment orders that are associated with the account. <p>A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.</p></p>\n",
-        "operationId": "GET /accounts/{accountId}/numbersAssignment",
+        "description": "Retrieve a list of the TelephoneNumbersAssignment orders that are associated with the account. A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.",
+        "summary": "List Telephone Numbers Assignment orders",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7665]:
```suggestion
        "summary": "List Telephone Numbers Assignment Orders",
```

File: site/specs/numbers.json [hunk 2]
```diff
@@ -7661,8 +7661,9 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieve a list of the TelephoneNumbersAssignment orders that are associated with the account. <p>A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.</p></p>\n",
-        "operationId": "GET /accounts/{accountId}/numbersAssignment",
+        "description": "Retrieve a list of the TelephoneNumbersAssignment orders that are associated with the account. A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7664]: In the new world we aren't marking things as required with a `*`.  Here's what it looks like now
image

File: site/specs/numbers.json [hunk 3]
```diff
@@ -7676,143 +7677,144 @@
...
+                  "PARTIAL",
+                  "FAILED"
+                ]
               }
             },
             "example": "PARTIAL"
           },
           {
             "name": "tn",
-            "description": "<p><b class=\"required\">*</b> A Telephone Number (TN) that is referenced in the order</p>\n",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7692]: Is this not actually required anymore?

File: site/specs/numbers.json [hunk 4]
```diff
@@ -7676,143 +7677,144 @@
...
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "ABCCorp12345"
           },
           {
             "name": "createdDateFrom",
-            "description": "<p>Checks the order's creation date against this value. Orders that have a creation date after this date will be included. Format is yyyy-MM-dd</p>\n",
+            "description": "Checks the order's creation date against this value. Orders that have a creation date after this date will be included. Format is yyyy-MM-dd</p>",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7718]: You deleted the open `<p>` tag, but not the closing one

File: site/specs/numbers.json [hunk 5]
```diff
@@ -7676,143 +7677,144 @@
...
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "2013-10-25"
           },
           {
             "name": "orderIdFragment",
-            "description": "<p><b class=\"required\">*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TelephoneNumbersAssignment Order.  This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.</p>\n",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7736]: Is this not actually required anymore?

File: site/specs/numbers.json [hunk 6]
```diff
@@ -7676,143 +7677,144 @@
...
                   }
                 }
               }
             }
           },
           "400": {
-            "description": "<p>Bad Request.  If any errors are found in the processing of the query string a 400 will be returned.  Note that parameters that are not recognized are not considered errors, and are just ignored.</p>\n"
+            "description": "Bad Request. If any errors are found in the processing of the query string a 400 will be returned.  Note that parameters that are not recognized are not considered errors, and are just ignored."
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7797]: Below you have a `<br>` after Bad Request.  Should we have that here too to be consistent?

File: site/specs/numbers.json [hunk 7]
```diff
@@ -7862,42 +7864,41 @@
...
-        "description": "<p>Retrieve information about a TelephoneNumbersAssignment order with specified ID.</p>\n",
-        "operationId": "GET /accounts/{accountId}/numbersAssignment/{orderid}",
+        "description": "Retrieve information about a TelephoneNumbersAssignment order with specified ID.",
+        "summary": "Fetching Telephone Numbers Assignment order status",
+        "operationId": "ReadTelephoneNumbersAssignmentOrder",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
+            "description": "Account id",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:7873]: I think everywhere else we referred to it as `User's account ID`

File: site/specs/numbers.json (hunk 3 above)
Comment (xxlaykxx) [reply site/specs/numbers.json:7692]: it was never required

File: site/specs/numbers.json [hunk 8]
```diff
@@ -7676,143 +7677,144 @@
...
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "9199918388"
           },
           {
             "name": "customerOrderId",
-            "description": "<p><b class=\"required\">*</b> The Customer Order ID is an ID assigned by the account owner to provide a reference number for the TelephoneNumbersAssignment order.</p>\n",
+            "description": "The Customer Order ID is an ID assigned by the account owner to provide a reference number for the TelephoneNumbersAssignment order.",
```
[the remark this answers is not in the document]
Comment (xxlaykxx) [reply site/specs/numbers.json:7708]: it was never required

File: site/specs/numbers.json (hunk 5 above)
Comment (xxlaykxx) [reply site/specs/numbers.json:7736]: it was never required

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**